dbld/rules

```diff
@@ -39,7 +39,6 @@ DOCKER_RUN_ARGS=-e USER_NAME_ON_HOST=$(shell whoami)	\
 	-e GRADLE_PROJECT_CACHE_DIR=/build/gradle-cache \
 	-e GRADLE_FLAGS=--build-cache \
 	$(if $(wildcard ${HOME}/.gitconfig),-v ${HOME}/.gitconfig:/build/.gitconfig)
-DOCKER_BUILD_ARGS=
```

Comment:
I wouldn't remove this from here, as we are using it in our image-% rule below this point. you can still override this value, or maybe we could use ?= to only assign the empty value if it's unset.

Reply:
the removal was intentional, having that over there will empty an already, externally pre-defined value, so you will not be able to pass it through the chain